docs/en/observability/synthetics-quickstart.asciidoc

```diff
@@ -21,144 +23,111 @@ and a packaged todo application with a custom suite of tests.
 
-WARNING: Running a Chrome browser requires elevated privileges.
-Synthetic monitoring scripts can escape the docker container.
-It is recommend to run your tests on a separate box.
-Do not run any scripts that you don’t trust.
+Run the script below to start running your synthetics tests. You'll need to insert your actual `cloud.id` and `cloud.auth` values to successfully index data to your cluster.
 
-[source,sh]
+// NOTE: We do NOT use <1> references in the below example, because they create whitespace after the trailing \
+// when copied into a shell, which creates mysterious errors when copy and pasting!
+[source,sh,subs="+attributes"]
 ----
-#...
 docker run \
   --rm \
   --name=heartbeat \
   --user=heartbeat \
-  --net=host \
-  --security-opt seccomp=seccomp_profile.json \ <1>
-  --volume="$(pwd)/heartbeat.docker.yml:/usr/share/heartbeat/heartbeat.yml:ro" \ <2>
-  --volume="$(pwd)/../../:/opt/elastic-synthetics:rw" \ <3>
-  $IMAGE \
-  --strict.perms=false -e \
-  $HEARTBEAT_ARGS
-#...
-----
-<1> Running a Chrome browser requires elevated privileges.
-Do not run any scripts that you don't trust.
-<2> Provides your `heartbeat.docker.yml` file as a volume.
-<3> Provides the `elastic-synthetics` repo as a volume.
-
-Use the following command to run the provided sample tests (or your own).
-Don't forget to update the example with your {es} credentials.
-
-// Right now we do not publish docker images for patch releases
-// Docker pull should reference only the .0 bugfix of the minor
-// We can accomplish this with the following attribute `{minor-version}.0`
-
-[source,sh,subs="attributes"]
-----
-sh run.sh {minor-version}.0 \
-  '-E cloud.id=<cloud-id>' \
-  '-E cloud.auth=elastic:<cloud-pass>'
+  --volume="$PWD/heartbeat.yml:/usr/share/heartbeat/heartbeat.yml:ro" \
+  --volume="$PWD:/opt/todos:ro" \
+  {synthetics-image} heartbeat -e \
+  -E cloud.id=cloud-id \
+  -E cloud.auth=elastic:cloud-pass
 ----
 
 If you aren't using {ecloud}, replace `-E cloud.id` and `-E cloud.auth` with your Elasticsearch hosts,
 username, and password:
 
 [source,sh,subs="attributes"]
 ----
-sh run.sh {minor-version}.0 \
-  '-E output.elasticsearch.hosts=["localhost:9200"]' \
-  '-E output.elasticsearch.username=elastic' \
-  '-E output.elasticsearch.password=changeme'
+docker run \
+  --rm \
+  --name=heartbeat \
+  --user=heartbeat \
+  --volume="$PWD/heartbeat.yml:/usr/share/heartbeat/heartbeat.yml:ro" \
+  --volume="$PWD:/opt/todos:ro" \
+  {synthetics-image} heartbeat -e \
+  -E output.elasticsearch.hosts=["localhost:9200"] \
+  -E output.elasticsearch.username=elastic \
+  -E output.elasticsearch.password=changeme
 ----
 
+Note the two `--volume` options which mount local directories into the container. The first mounts the `heartbeat.yml` file you've configured,
+into the expected location for `heartbeat.yml` for heartbeat, the second mounts the suite into the VM in the same location listed in the `source`
+section of our `heartbeat.yml`.
```

Comment:
I'm struggling to parse the long second sentence. What about something like this?

```suggestion
Note the two `--volume` options, which mount local directories into the container. The first mounts the `heartbeat.yml` file configured above,
into Heartbeat's expected location for `heartbeat.yml`. The second mounts the test suite into the VM in the same location listed in the `source`
section of our `heartbeat.yml`.
```